### ISSUE TITLE
JavaDoc contradictions

### ISSUE THREAD

**Issue by TreyWurm** [davideas/FlexibleAdapter#728]
JavaDoc for addItem()/addItems() states that when passing a negative position it will be appended to the end. On `eu.davidea:flexible-adapter:5.1.0` you got following annotation
`@IntRange(from = 0)` which prevents passing negative positions

**Comment by davideas** [comment]
@TreyWurm, yes you are right, however that is just a lint suggestion, you can disable, you can pass any value.
I will fix it for the next version. Thanks.
